CoreConversationView/src/au/gov/asd/tac/constellation/views/conversationview/ConversationBox.java

```diff
@@ -121,7 +121,7 @@ public final class ConversationBox extends StackPane {
     private int searchCount;
 
     private static final String FOUND_TEXT = "Showing ";
-    private static final String FOUND_PASS_COLOR = "-fx-text-fill: yellow;";
+    private static final String FOUND_PASS_COLOR = JavafxStyleManager.isDarkTheme() ? "-fx-text-fill: yellow;" : "-fx-text-fill: blue;";
```

Comment:
wondering whether maybe some sort of dark yellow may be more complimentary than blue?

Reply:
Tested with many shades of yellow but they don't look quite right in light mode in my opinion. They need to be dark enough to provide contrast and the colour ends up looking like a dark mustard color.
![image](https://github.com/constellation-app/constellation/assets/139201041/001c1b36-667b-4674-8edc-c93c2eee2bab)
![image](https://github.com/constellation-app/constellation/assets/139201041/be513661-ba0b-4fb9-8c12-72d1ed7486b3)
![image](https://github.com/constellation-app/constellation/assets/139201041/e4d4e789-2faa-4a88-a4eb-e2e2cf80a8b2)

I also experimented with using grey for the buttons and I have to agree with Delphinus in that the application is a bit harder to navigate with grey buttons instead of being colored. Most of the views also look very bland in a monochrome color scheme, but am happy to experiment further if requested.

![image](https://github.com/constellation-app/constellation/assets/139201041/0fe21fa4-61f6-47b0-8761-ee807666f6d6)
![image](https://github.com/constellation-app/constellation/assets/139201041/fe126e38-2581-413e-80f4-c2e53b9a49f3)
![image](https://github.com/constellation-app/constellation/assets/139201041/169b6c50-8a57-4664-9a2a-be27c25c8851)
![image](https://github.com/constellation-app/constellation/assets/139201041/0b7fd7c0-6c1f-4f21-bc47-24e25288ad51)